coalib/results/result_actions/OpenEditorAction.py

```diff
@@ -96,6 +96,10 @@
         'args': '--new-window',
         'gui': True
     },
+    'brackets': {
```

Comment:
alpha order please

Reply:
@jayvdb I checked, brackets does not support jumping to a line number.